README.md

```diff
@@ -1,27 +1,37 @@
 # Project Name
```

Comment:
# You didn't make changes to the project name. It should be change to FCC PIANO. 
## That's the only thing  that needs correcting here except that it all good